content/using-sagemaker-managed-spot-training/cleanup.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Cleanup"
```

Comment:
I think there's a need for a final page after the Cleanup, maybe with links for further learning or videos. it might also be confusing that when clicking the next page button participants just get to the next workshop